TestingLowerBounds/ForMathlib/RnDeriv.lean

```diff
@@ -269,6 +269,11 @@ lemma trim_withDensity (hm : m ≤ mα) [SigmaFinite μ] {f : α → ℝ≥0∞}
 
 end Trim
 
+lemma rnDeriv_toReal_ne_zero [SigmaFinite μ] [SigmaFinite ν] (hμν : μ ≪ ν) :
+    ∀ᵐ x ∂μ, (μ.rnDeriv ν x).toReal ≠ 0 := by
```

Comment:
You should strengthen this one to a conclusion `0 < ...` instead of `... ≠ 0`

Reply:
Done